mteb/tasks/Classification/ara/AJGT.py

```diff
@@ -16,7 +19,7 @@ class AJGT(AbsTaskClassification):
         type="Classification",
         category="s2s",
         modalities=["text"],
-        eval_splits=["train"],
+        eval_splits=["test"],
```

Comment:
I would only say "test" here if there was actually a "test" set on the hub, otherwise it will probably cause some confusion.